src/pudl/ferc_to_sqlite/__init__.py

```diff
@@ -14,11 +14,23 @@
 
 @graph
 def ferc_to_sqlite():
-    """Clone the FERC Form 1 FoxPro database and XBRL filings into SQLite."""
+    """Clone the FERC FoxPro databases and XBRL filings into SQLite."""
     dbf2sqlite()
     xbrl2sqlite()
 
 
+@graph
+def ferc_to_sqlite_dbf_only():
+    """Clone the FERC FoxPro databases into SQLite."""
+    dbf2sqlite()
+
+
+@graph
+def ferc_to_sqlite_xbrl_only():
+    """Clone the FERC XBRL databases into SQLite."""
+    xbrl2sqlite()
+
+
```

Comment:
Would it be easy to break these out both by data format and by individual forms?  I think in development we'd typically be working with a single form and format at a time, and being able to just regenerate that one SQLite DB in isolation would be convenient.

I guess you've created this possibility with the "disabled" option.